app/controllers/users.js

```diff
@@ -14,3 +17,27 @@ exports.create = (req, res, next) => {
     .then(user => res.status(201).send(user))
     .catch(reason => next(errors.defaultError(`Database error - ${reason}`)));
 };
+
+exports.authenticate = (req, res, next) => {
+  User.findOne({ where: { email: req.body.email } })
```

Comment:
return this promise

Reply:
👍 Being this the only expression in the function isn't this unnecessary?